native/cocos/platform/android/adpf_manager.h

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright 2022 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef ADPF_MANAGER_H_
+#define ADPF_MANAGER_H_
```

Comment:
Please fix the clangtidy error. Why not just use `#pragma once`.

Reply:
These files are base on the demo from Google

Reply:
We can change it to obey the engine coding style.

Reply:
I think we should minimize the changes to make it easier for future synchronization.

Reply:
I don't think it is a big change here.

Reply:
The coding standards of these two files differ from our existing code, including naming conventions. If we were to fully conform to our current code standards, the changes would be significant.

Reply:
Ok, sounds good. Currently, just this line causes clang tidy error.